math/src/test/kotlin/ktx/math/rangesTest.kt

```diff
@@ -232,4 +235,18 @@ class RangesTest {
     }
     assert(values.all { it in range })
   }
+
+  @Test
+  fun `should interpolate`() {
+    val start = Random.nextFloat()
+    val end = Random.nextFloat()
+    val progress = Random.nextFloat()
+    val range = start..end
+    val expectedLinear = (end - start) * progress + start
```

Comment:
Afaik it's exactly the same logic as the actual implementation, so it will likely return the same value. I generally avoid tests like this and rely on predefined values instead, e.g. start = 1, end = 3, progress = 0.5, expected result = 2.

Reply:
The same applies to the test with interpolation.